### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by mckoss** [reskillamericans/website#31]
I have not been adding Aria tags to the website. So we have a pretty poor accessibility rating right now.

@sylvercodez do you have experience with aria?

Is so, that might be a good project for you to go through the site and fix the markup so that it is accessible.

I want to be sure that we have some sort of testing and make sure that it's done properly. I don't normally have a screen reader set up, so I didn't feel confident just adding markup without knowing I was doing it properly.

**Comment by sylvercodez** [comment]
I will handle this thank you..

**Comment by sylvercodez** [comment]
I have very little experience using aria but I think it's time to brush it over... I will do this

**Comment by mckoss** [comment]
I made a pass using html-validate and fixed all the errors (mostly missing alt text).  Lighthouse score now looks good.

![image](https://user-images.githubusercontent.com/82750/149057308-bec023a8-400c-452c-97c3-afe43cb852f9.png)

5e787b93900c8b5b45b4a771278822903f13eac5
